Src/main.c

```diff
@@ -1832,6 +1839,12 @@ int main(void)
         }
 #endif
 
+#ifdef MCU_CH32V203
```

Comment:
we should add a NEED_INPUT_READY define for F031 and V203, instead of copying the code

Reply:
Yes, NEED_INPUT_READY would be bettter,  it was distinguished by name before.